util/content.go

```diff
@@ -99,6 +99,8 @@ type Content struct {
 	// them (unlike with aggregates)
 	DagSplit  bool `json:"dagSplit"`
 	SplitFrom uint `json:"splitFrom"`
+
+	FailedSanityCheck bool `json:"-"`
```

Comment:
Introducing a new column in a massive table like Content needs to have a roll forward and back script. We will need to execute this SQL script on API node first before we deploy the app.

Reply:
I have changed it to use a new table